web_disable_export_group/static/src/js/disable_export_group.js

```diff
@@ -0,0 +1,37 @@
+/* Copyright 2016 Onestein
+   Copyright 2018 Tecnativa - David Vidal
+   License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl). */
+odoo.define("web_disable_export_group", function(require) {
+"use strict";
+
+    var core = require("web.core");
+    var Widget = require("web.Widget");
+    var Sidebar = require("web.Sidebar");
+    var session = require("web.session");
+    var _t = core._t;
+
+    Widget.include({
+        init: function (parent) {
+            this._super(parent);
+            session.user_has_group('web_disable_export_group.group_export_data').then(function(can_export) {
+                if (can_export) {
+                    session.group_export_data = true;
+                }
+            });
+        },
+    });
+
+    Sidebar.include({
+        _addItems: function (sectionCode, items) {
+            if (!session.is_superuser && sectionCode === 'other' && items.length && !session.group_export_data) {
+                var export_label = _t("Export");
+                for (var i = 0; i < items.length; i++) {
```

Comment:
Here can use "underscore"... 
```items = _.reject(items, {label:_t("Export")});```

But need pass 'sectionCode' and 'items' to super... or use "arguments[1]" instead of "items" because underscore generate a new list (reference is changed).